resources/css/game-category.css

```diff
@@ -0,0 +1,48 @@
+.game-category {
```

Comment:
generalize as `.label` to be reusabe in other areas.
or `.achievement-set-category` to match the upcoming schema changes

Reply:
Added a generic `.tag` (in `tag.css`) and an `achievement-set.css` file with specific style for categories and subsets.